File: iniciante/javascript/1060.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

function positivo(a){
    if(a > 0 ){
        return 1
    }
    return 0
}

let countEntradas = 0
let countPositivo = 0

while(countEntradas < 6){
    let entrada = lines.shift()
    countPositivo += positivo(entrada)
    countEntradas++
}

console.log(countPositivo+' valores positivos')

File: iniciante/python/1060.py
# -*- coding: utf-8 -*-

count = 0
count_pares = 0
numeros = []

while count < 6:
    numeros.append(input())
    if '.' in numeros[count]:
        numeros[count] = float(numeros[count])
    else:
        numeros[count] = int(numeros[count])
    if numeros[count] > 0:
        count_pares += 1
    count += 1

print(f'{count_pares} valores positivos')
